tests/components/velbus/test_init.py

```diff
@@ -54,3 +55,17 @@ async def test_device_identifier_migration(
     assert device_entry.manufacturer == "Velleman"
     assert device_entry.model == "module_type_name"
     assert device_entry.sw_version == "module_sw_version"
+
+
+async def test_migrate_config_entry(hass: HomeAssistant):
+    """Test successful migration of entry data."""
+    legacy_config = {CONF_NAME: "fake_name", CONF_PORT: "1.2.3.4:5678"}
+    entry = MockConfigEntry(domain=DOMAIN, data=legacy_config)
+
+    assert entry.data == legacy_config
+    assert entry.version == 1
+
+    await entry.async_migrate(hass)
```

Comment:
Don't call the migrate method directly. Set up the legacy config entry and let it be migrated. Use `hass.config_entries.async_setup`.